packages/isaac-typescript-definitions-repentogon/src/types/classes/StageTransition.d.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * This class is for REPENTOGON, an exe-hack which expands the modding API.
+ *
+ * @see https://repentogon.com/index.html
+ * @noSelf
+ */
+declare namespace StageTransition {
+  /**
+   * Returns whether the stage transition screen displays Isaac's head moving from one stage to the
+   * other.
+   */
+  function GetSameStage(): boolean;
+
+  // eslint-disable-next-line isaacscript/complete-sentences-jsdoc
```

Comment:
its crazy to me that you would disable the linter instead of fixing the typo (?)